README.md

```diff
@@ -97,178 +91,148 @@
       </ul>
     </li>
   </ul>
-
   <!-- INSTALLATION -->
   <h2 id="installation"><span style="font-variant: small-caps;">Installation</span></h2>
-
   <!-- INSTALLATION/EWON SETUP -->
   <h3 id="ewon-setup"><span style="font-variant: small-caps;">Ewon Setup</span></h3>
   <p><em>During Ewon setup, you’ll need access to your Ewon device’s webpage. For information on accessing an Ewon device’s webpage, please refer to <a href="#accessing-an-ewon-devices-webpage">Appendix 6.3, Accessing an Ewon Device’s Webpage</a>.</em></p>
-
   <p><strong>Step 1:</strong> Using your Ewon device’s webpage, open the user account settings by selecting <code>Setup</code> &gt; <code>Users</code> in the menu on the left.</p>
```

Comment:
same with all these really. dont white spaces in html not matter when it gets rendered? I dont remember

Reply:
maybe do a white space commit